include/exadg/fluid_structure_interaction/precice/coupling_base.h

```diff
@@ -0,0 +1,259 @@
+/*  ______________________________________________________________________
+ *
+ *  ExaDG - High-Order Discontinuous Galerkin for the Exa-Scale
+ *
+ *  Copyright (C) 2022 by the ExaDG authors
+ *
+ *  This program is free software: you can redistribute it and/or modify
+ *  it under the terms of the GNU General Public License as published by
+ *  the Free Software Foundation, either version 3 of the License, or
+ *  (at your option) any later version.
+ *
+ *  This program is distributed in the hope that it will be useful,
+ *  but WITHOUT ANY WARRANTY; without even the implied warranty of
+ *  MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ *  GNU General Public License for more details.
+ *
+ *  You should have received a copy of the GNU General Public License
+ *  along with this program.  If not, see <https://www.gnu.org/licenses/>.
+ *  ______________________________________________________________________
+ */
+
+#ifndef INCLUDE_EXADG_FLUID_STRUCTURE_INTERACTION_PRECICE_COUPLING_BASE_H_
+#define INCLUDE_EXADG_FLUID_STRUCTURE_INTERACTION_PRECICE_COUPLING_BASE_H_
+
+
+#include <deal.II/base/conditional_ostream.h>
+#include <deal.II/base/vectorization.h>
+
+#include <deal.II/matrix_free/matrix_free.h>
+
+#include <exadg/matrix_free/integrators.h>
+#include <precice/SolverInterface.hpp>
```

Comment:
Just a minor comment here: Please group together the different projects deal.II, ExaDG, and precice, with empty lines in between.